edit-post/components/header/index.js

```diff
@@ -78,13 +78,13 @@ function Header( {
 
 export default connect(
 	( state ) => ( {
-		isGeneralSidebarEditorOpen: getOpenedGeneralSidebar( state ) === 'editor',
+		isEditorSidebarOpen: isEditorSidebarOpened( state ),
 		isPublishSidebarOpen: isPublishSidebarOpened( state ),
 		hasActiveMetaboxes: hasMetaBoxes( state ),
 		isSaving: isSavingMetaBoxes( state ),
 	} ),
 	{
-		onOpenGeneralSidebar: () => openGeneralSidebar( 'editor' ),
+		onOpenGeneralSidebar: openGeneralSidebar.bind( 'edit-post/document' ),
```

Comment:
Should the argument be the second parameter?

Reply:
Yeah, probably is better idea to stick with the arrow function 😃